include/SFML/Window/GlResource.hpp

```diff
@@ -53,6 +66,12 @@ class SFML_WINDOW_API GlResource
     ////////////////////////////////////////////////////////////
     ~GlResource();
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Move constructor
+    ///
+    ////////////////////////////////////////////////////////////
+    GlResource(GlResource&&) noexcept = default;
```

Comment:
`default` does nothing here. This is not correct. We need to steal ownership of the implicit link with `GlContext` so that reference count remains correct.

Reply:
`GlResource` doesn't own any `GlContext`. Its constructor and destructor only create global variable of `sharedContext` if not available and it delete them if `recourseCount` reach 0. the `GlContext` lives in Window class that is moved.